app/src/main/java/tech/pauly/findapet/discover/AnimalListItemViewModel.java

```diff
@@ -0,0 +1,53 @@
+package tech.pauly.findapet.discover;
+
+import android.databinding.ObservableField;
+
+import java.util.List;
+
+import tech.pauly.findapet.data.models.Animal;
+import tech.pauly.findapet.data.models.Media;
+import tech.pauly.findapet.data.models.Photo;
+import tech.pauly.findapet.data.models.PhotoSize;
+
+public class AnimalListItemViewModel {
+
+    public ObservableField<String> name = new ObservableField<>("");
+    public ObservableField<String> imageUrl = new ObservableField<>("");
+    public ObservableField<String> age = new ObservableField<>("");
+    public ObservableField<String> breeds = new ObservableField<>("");
+
+    public AnimalListItemViewModel(Animal animal) {
+        name.set(animal.getName());
+        age.set(animal.getAge().toString());
+
+        setBreeds(animal.getBreedList());
+        setPhoto(animal.getMedia());
+    }
+
+    private void setBreeds(List<String> breedList) {
+        StringBuilder breedString = new StringBuilder();
+        for (int i = 0; i < breedList.size(); i++) {
```

Comment:
Probably want to cap this eventually, but I'm interested to see how many breeds people put, I've seen 3 max so far.